### ISSUE TITLE
[D365 Omnichannel - Channel Experiences][Web][Chat][ScreenReader]: Screen reader is announcing “Bot JO said” for both the profile icons (BO and JO)

### ISSUE THREAD

**Issue by Kjasnani7** [microsoft/BotFramework-WebChat#5803]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader users (blind/low‑vision): Cannot determine whether responses are coming from the bot whom.
Users with cognitive disabilities: Incorrect speaker identification increases cognitive load and makes live agent support harder to follow.

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA, Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

ARIA roles

### What is the public URL for the website?

https://cesqamancscfacsppeorg01.crm10.dynamics.com/

### How to reproduce the issue?

1. Launch the application URL: https://cesqamancscfacsppeorg01.crm10.dynamics.com/and login with appropriate credentials.
2. TAB to the "Customer Service admin center" application and press ENTER.
3. TAB to "Workstreams" and press ENTER.
4. TAB to any workstream of type "messaging" and press ENTER.
5. TAB to "Edit" button and press ENTER.
6. "Let's chat" window will get opened.
7. Turn on Narrator and navigating using DOWN arrow key.
8. After the agent joined, screen reader is announcing “Bot JO said” for both the profile icons (BO and JO)

### What do you expect?

Screen reader should not announce “Bot JO said” for both the profile icons (BO and JO)

### What actually happened?

Screen reader is announcing “Bot JO said” for both the profile icons (BO and JO).

### Do you have any screenshots or recordings to repro the issue?

<img width="1272" height="711" alt="Image" src="https://github.com/user-attachments/assets/dd914717-0768-42fc-9091-2d13d87f1d8c" />

https://github.com/user-attachments/assets/b4a14f23-f0ca-49b9-87f6-e56abefe029b

https://github.com/user-attachments/assets/5aef71e8-89f2-47ec-8e92-0ff265e63deb

### MAS reference

https://aka.ms/MAS4.1.2

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

### Adaptive Card JSON

```json

```

**Comment by compulim** [comment]
@Kjasnani7 can you describe what we should *actually* expecting?

You mentioned "Screen reader should not announce “Bot JO said” for both the profile icons (BO and JO)" and it doesn't help us understand the problem.

**Comment by Kjasnani7** [comment]
Ideally, it should announce the initial with the respective button. For Bot BO, it should announce Bot BO, and for Bot Jo, it should announce Bot Jo. If we use a screenshot for reference, it should announce accordingly.
"Bot BO said: Okay, let me connect you with a live agent.
BO
Bot BO said:
clickable Okay, let me connect you with a live agent.
Bot- 1:17 PM
blank
John has joined the chat
blank
JO
Bot JO said:"l'd be happy to help you update your account.
